app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -29,35 +29,42 @@ def jwt_identity
 
         # Checks if variable that defined from which field in decoded token to get the id is configured
         def identity_available
-          return @identity_available unless @identity_available.nil?
+          return @identity_available if defined?(@identity_available)
+
           @identity_available ||= identity_field_variable.present?
```

Comment:
i think that you can remove the memoization
```suggestion
          @identity_available = identity_field_variable.present?
```

Reply:
Done